compiler/src/dotty/tools/dotc/core/tasty/TastyString.scala

```diff
@@ -0,0 +1,20 @@
+package dotty.tools.dotc.core.tasty
+
+/** Utils for String representation of TASTY */
+object TastyString {
+
+  /** Decode the TASTY String into TASTY bytes */
+  def stringToTasty(str: String): Array[Byte] = { // TODO factor out this and tastyToString
```

Comment:
`str.getBytes`?

Reply:
No, the string encoding messes up the bytes